crates/cli/src/config.rs

```diff
@@ -169,8 +169,8 @@ impl RawConfig {
             ecdsa_public_key: None,
         };
         RawConfig {
-            default_server: local.nickname.clone(),
-            server_configs: vec![local, maincloud],
```

Comment:
I would like to get @drogus and @jdetter's thoughts on the above.

Reply:
The last part is actually not true. Upgrading your CLI will not change your default server.